misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/NamedCompoundVariable.java

```diff
@@ -66,6 +66,12 @@ public Value getChildByName(String name) throws DebugVariableException {
         }
 
         if (!namedChildVariables.containsKey(name)) {
+            for (Map.Entry<String, Value> childVariable : namedChildVariables.entrySet()) {
+                String escaped = childVariable.getKey().replaceAll("\\$0092(\\$0092)?", "$1");
```

Comment:
imo, using a constant fro `$0092` will improve the readability of the code.
It is not mandatory because it is using only in here.